platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/typesafe/TypeSafeProjectAccessorsIntegrationTest.groovy

```diff
@@ -18,11 +18,8 @@ package org.gradle.integtests.resolve.typesafe
 
 import org.gradle.integtests.fixtures.FeaturePreviewsFixture
 import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
-import org.gradle.integtests.fixtures.resolve.ResolveTestFixture
 
 class TypeSafeProjectAccessorsIntegrationTest extends AbstractTypeSafeProjectAccessorsIntegrationTest {
-    def resolve = new ResolveTestFixture(buildFile)
```

Comment:
What are these changes about?

Reply:
The tests were like this in the past. We had to make a bunch of changes to the tests since we could no longer call `.dependencyProject.path` on the typesafe project accessors anymore. The workaround was to perform a resolution and check the graph at the end. 

However, now that `path` is exposed, we can simplify these tests back to the way they were